crates/rust-analyzer/src/config.rs

```diff
@@ -56,6 +56,8 @@ config_data! {
     struct ConfigData {
         /// Placeholder expression to use for missing expressions in assists.
         assist_expressionFillDefault: ExprFillDefaultDef              = "\"todo\"",
+        /// Specify type of missing expression inside generated todo macro invocation.
+        assist_expressionFillTypeHints_enable: bool = "true",
```

Comment:
This should be disabled by default, this is a rather noisy setting so I don't think it makes a good default if enabled.

Reply:
Done :)